src/io/flutter/logging/FlutterLogView.java

```diff
@@ -405,19 +410,43 @@ public void disableIfNeeded() {
     }
 
     @Override
-    public void actionPerformed(AnActionEvent e) {
+    public void actionPerformed(@NotNull AnActionEvent e) {
       logTree.sendSelectedLogsToClipboard();
     }
   }
 
+  class DataPanel extends JEditorPane {
+    DataPanel() {
+      setEditable(false);
+    }
+
+    public void update() {
+      final StringBuilder sb = new StringBuilder();
+      for (FlutterLogTree.FlutterEventNode node : logTree.getSelectedNodes()) {
+        final String data = node.entry.getData();
+        if (data != null && !Objects.equals(data, "null")) {
```

Comment:
showing multiple selected nodes is a bit surprising to me but maybe it is helpful. I don't think any of other other master-detail views allow multiple selection like that... at least not yet.